quiche/quic/core/quic_packet_writer.h

```diff
@@ -170,8 +170,8 @@ class QUICHE_EXPORT QuicPacketWriter {
   // - Return WRITE_STATUS_BLOCKED if the underlying socket is blocked while
   //   sending. Some packets may have been sent, packets not sent will stay in
   //   the internal buffer.
-  // - Return a status >= WRITE_STATUS_ERROR if an error was encuontered while
-  //   sending. As this is not a re-tryable error, any batched packets which
+  // - Return a status >= WRITE_STATUS_ERROR if an error was encountered while
+  //   sending. As this is not a retryable error, any batched packets which
```

Comment:
retriable also appears twice, but retryable is used and again, I'm not trying to force a dominant spelling.

The world is very on the fence for this:
https://english.stackexchange.com/questions/305273/retriable-or-retryable